data/GA/incentives.json

```diff
@@ -0,0 +1,170 @@
+[
+  {
+    "id": "GA-1",
+    "authority_type": "utility",
+    "authority": "ga-georgia-power",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "ga_georgiaPower_homeEnergyImprovementProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250,
+      "maximum": 250
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Rebate for 50% of cost up to $250 for attic insulation. Addition of R-38 or greater required."
```

Comment:
This should be type percentage with number 0.5, but the 250 maximum is correct.

Same true for all of the above (these are errors in the original spreadsheet).